BuddySave/FileManagement/RollingBackups.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BuddySave.Core.Models;
+
+namespace BuddySave.FileManagement;
+
+public class RollingBackups : IRollingBackups
```

Comment:
Alternative implementation idea: RollingBackups could be implemented in a way that you tell it to add a backup and it manages itself to check if the old ones need to be removed.

Reply:
They could, I was thinking of that, but then not sure, what BackupManager does.

Reply:
Moved logic into RollingBackups.Add()